modules/claims_api/lib/bgs_service/redis/find_poas_service.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+require 'common/models/concerns/cache_aside'
+require 'bgs_service/standard_data_web_service'
+require 'bgs_service/redis/find_poas_response'
+
+module ClaimsApi
+  class FindPOAsService < ::Common::RedisStore
+    include ::Common::CacheAside
+
+    redis_config_key :bgs_find_poas_response
+
+    def response
+      @response ||= response_from_redis_or_service.response
+    end
+
+    private
+
+    def todays_date
+      Time.zone.now.to_date.to_s
+    end
+
+    def response_from_redis_or_service
+      do_cached_with(key: todays_date) do
```

Comment:
e.g., `2024-09-06`

We'll want to be careful with our (future) daily cron job. Depending on when the job runs, we may cache under yesterday's date, meaning the first unlucky user to call our service that day will hit BGS instead of the cache.